scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -20,15 +20,28 @@ class SensorVersionsFailure(Exception):
 
 gkecluster=GKECluster("compat-test")
 
+if len(chart_versions) > 1:
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[0]
+
 failing_sensor_versions = []
 for version in chart_versions:
     os.environ["SENSOR_CHART_VERSION"] = version
     try:
         make_compatibility_test_runner(cluster=gkecluster).run()
     except Exception:
-        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version}")
+        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version} and central version {chart_versions[0]}")
```

Comment:
[nit] Usually, central version goes before sensor version in all places. Could you please stick to that?

Reply:
Done